### PR TITLE
add simple freebsd templale

### PR DESCRIPTION
**Pull request by takekazuomi**
### Changelog
* add freebsd template 

### Description of the change
Congratulations FreeBSD, Welcome, to Azure
Now add a FreeBSD template

This template is based on the 101-vm-simple-linux.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@takekazuomi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

File: 101-vm-simple-freebsd/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    },
+    "freeBSDOSVersion": {
```
Comment (singhkay) [inline 101-vm-simple-freebsd/azuredeploy.json]: If the allowed and default value is same, then this should be a variable

File: 101-vm-simple-freebsd/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a simple FreeBSD VM in West US",
```
Comment (singhkay) [inline 101-vm-simple-freebsd/metadata.json]: Location is the resource group location so the name is not accurate.

**Comment by singhkay** [comment]
@takekazuomi Thank you for taking this on! Left a few comments

**Comment by takekazuomi** [comment]
@singhkay 
I have two changes.
1. current FreeBSD is only one sku. Thus was the variable from the parameter.
2. Change the comments about the location

Thank you

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**